packages/components/tests/dummy/app/styles/pages/db-tabs.scss

```diff
@@ -1,6 +1 @@
 // TABS
-
-img.dummy-figma-docs {
```

Comment:
[issue] this file isn't getting included in the dummy app's style sheet so these styles are not being applied to the element.

Reply:
I'll just delete the file since I'm not currently using it, thanks.